src/lib/layout/hexagons/TeamMemberHexagon.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+  import Hexagon from './Hexagon.svelte';
+  import Icon from '$lib/component/Icon.svelte';
+  import Icons from '$lib/component/Icons';
+  import type { TeamMember } from '@/model/TeamMember';
+
+  export let orientation: 'horizontal' | 'vertical';
+  export let teamMember: TeamMember;
+</script>
+
+<Hexagon {orientation}>
```

Comment:
If vertical hexagons don't work properly and are not needed maybe remove them from being an option to avoid future mistakes :smile: maybe you can create a low priority issue (not sure if it is beginner friendly) to address vertical orientations when those are needed ^^